<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/icons/icons/lion.png</normaloff>:/icons/icons/lion.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54, 141, 197);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rsc.qrc">:/images/images/beach.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>312</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_Number_LineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid rgb(157, 143, 132);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 2px;
	background-color:rgb(216, 197, 181);
}
</string>
   </property>
   <property name="placeholderText">
    <string>ID NUMBER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetCheckIn_PushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.397727 rgba(68, 164, 250, 255), stop:1 rgba(255, 255, 255, 255));
	
	color: rgba(255, 255, 255, 210);
	border-radius: 12px;
}


QPushButton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	
}</string>
   </property>
   <property name="text">
    <string>Set Checked In Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetCheckOut_PushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.397727 rgba(68, 164, 250, 255), stop:1 rgba(255, 255, 255, 255));
	
	color: rgba(255, 255, 255, 210);
	border-radius: 12px;
}


QPushButton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	
}</string>
   </property>
   <property name="text">
    <string>Set Checked Out Date</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
